proposals/wg-image-compatibility.md

```diff
@@ -0,0 +1,114 @@
+
+For bare-metal and cloud operating systems can use different kernel versions with different configurations.
+Different operating system distributions may integrate different devices and their corresponding drivers, even for the same device different versions of drivers may be used.
+Therefore, different operating system distributions are not identical.
+Even within the same OS distro, different releases are also not identical.
+
+An image compatibility would help container image authors describe compatibility requirements in a standard way.
+The specification will be uploaded with the image to the image registry.
+This makes hard container compatibility requirements discoverable, programmable, and will support different consumers and cover use cases where the application requires a specific compatible environment.
+
+For further reference please follow:
+* [OCI Image Compatibility - the spec reasoning](https://docs.google.com/presentation/d/1F9GnCm2sULuyTJ5BEFZlL8Qjab81DK7g9Oy6qBfe5Qs/edit?usp=sharing)
+* [Revisit container and host OS compatibility](https://docs.google.com/document/d/1lzwh8DGMu5vXXHwJmnewYIMffkcOEvH8owX4UYjRcw0/edit#heading=h.fcxt9vheg92c)
+
+## Scope
+
+* Define image compatibility that describes special host OS requirements of containerized application.
+* The compatibility should describe container requirements for Linux, illumos, FreeBSD and Windows.
+* The final list of supported fields (like kernel modules, configuration, out-of-tree drivers etc.) is to be determined by the working group for each supported OS.
+* An appropriate solution for expressing compatibility should be determined by the working group.
+
+## Out of Scope
+
+* Change images build or container execution flow.
+* Introduce a new hard requirement to build images or run containers.
+* Cover applications ABI compatibility.
+
+## Intended work product
+
+* The result of this WG should be extended or a new OCI specification that allows containers to describe host OS requirements.
+* The specification will provide a much better experience for users to find out about container requirements against host OS, if those exist.
+* The specification will provide programmatic data for various OS operators that could influence configuration of nodes.
+
+## Proposed Owners
+
+The following have agreed to participate in the working group, review progress, report progress back to the OCI community, and present the results to the TOB once the working group has completed its objectives.
+
+* Marcin Franczyk (@mfranczy)
+* Joe Huang (@chaoyihuang)
+* Zvonko Kaiser (@zvonkok)
+* Till Wegmüller (@toasterson)
+* Toru Komatsu (@utam0k)
+* B. Dewayne Branch (@GeoEducator)
+* Samuel McGraw (@smcgrawDotNet)
+* Vanessa Sochat (@vsoch)
+* Brandon Mitchell (@sudo-bmitch)
+* Aleksa Sarai (@cyphar)
```

Comment:
Since I might have issues attending, can we also add @dirkmueller to the list? He's involved with the BCI and image-building stuff we have at SUSE/openSUSE.

Reply:
Sure.

Reply:
Done.